korge-foundation/src@jvmAndroid/korlibs/event/_Datastructure_event.jvmAndroid.kt

```diff
@@ -1,6 +1,6 @@
 @file:Suppress("PackageDirectoryMismatch")
 
-package korlibs.datastructure.event
+package korlibs.event
```

Comment:
This is wrong. It should be `korlibs.datastructure.event` even if it is in the `korlibs-foundation` module. Otherwise we would be breaking compatibility. And we don't want that in this PR